types/is-regex/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-regex 1.0
+// Project: https://github.com/ljharb/is-regex
+// Definitions by: Jordan Harband <https://github.com/ljharb>
+//                 ExE Boss <https://github.com/ExE-Boss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Returns whether the value is a valid `RegExp` object with the `[[RegExpMatcher]]` internal slot.
+ */
+declare function isRegex(value: any): value is RegExp;
```

Comment:
```suggestion
declare function isRegex(value: unknown): value is RegExp;
```

Reply:
This is to allow using these types in **TypeScript 2.x**.

Reply:
DT already requires TypeScript 3.0.